.env.example

```diff
@@ -28,11 +28,12 @@ MONGO_SSL_CA=/tls/cacert
 MONGO_SSL_CLIENT_CERT_KEY=/tls/combined
 
 # rabbitmq
-RABBITMQ_PASSWORD=admin
-RABBITMQ_USERNAME=admin
+BROKER_PASSWORD=admin
+BROKER_USERNAME=admin
 BROKER_HOST=messagebroker
 BROKER_PORT=5672
 BROKER_PORT_SSL=5671
+BROKER_SSL=false
```

Comment:
not sure I see the reason of defaulting this here, when in `conf.py` we default to `False`
and in` docker-compose-tls.yml` on line 60 we hard code the value